src/linagora.esn.calendar/app/index.pug

```diff
@@ -2,7 +2,7 @@
   .col-xs-12(ng-if="alertMessage")
     span.calendar-error-message {{alertMessage}}
   .calendar-view
-    .col-md-3.col-lg-2.hidden-xs.hidden-sm
+    .col-md-3.col-lg-2.hidden-xs.hidden-sm(ng-if='hideSidebar')
```

Comment:
so this here means show the `cal-sidebar` component when the `hideSidebar` is true.

I suggest renaming `hideSidebar` to `showSidebar` cause it makes more sense when reading. or just invert it to `ng-if='!hideSidebar'` and invert the initial value in your run function:

```js
.run(function($rootScope) {
  $rootScope.hideSidebar = false; // Shown by default
})

```